renovate.json

```diff
@@ -31,6 +31,10 @@
 			"packageName": "typescript",
 			"prPriority": 1
 		},
+		{
+			"packagePatterns": [ "^@types/" ],
+			"prPriority": 1
```

Comment:
Maybe there should be also `groupName` field that tells Renovate to group the packages into one update and give the group a name?

Reply:
Good plan, added `Type definitions`.